frontend/ui/elements/waveform_level.lua

```diff
@@ -0,0 +1,36 @@
+local _ = require("gettext")
+local Device = require("device")
+local InfoMessage = require("ui/widget/infomessage")
+local UIManager = require("ui/uimanager")
+local Screen = Device.screen
+local T = require("ffi/util").template
+
+local items = {}
+for i=0, Screen.wf_level_max do
+    local info
+    if i == 0 then
+        info = _("Level 0: High quality, slow")
```

Comment:
Maybe something like this?
```suggestion
        info = _("Level 0: high quality, slowest")
```